zio-config/src/main/scala/zio/config/Config.scala

```diff
@@ -24,6 +27,8 @@ sealed trait Config[A] {
 
 object Config {
 
+  final case class Pure[A](a: A) extends Config[A]
```

Comment:
Be sure to have `Config.succeed` method for this constructor.

Reply:
@jdegoes  This is done